cellprofiler/modules/savecroppedobjects.py

```diff
@@ -51,6 +50,17 @@ def create_settings(self):
             value=cellprofiler.setting.DEFAULT_OUTPUT_FOLDER_NAME
         )
 
+        self.file_format = cellprofiler.setting.Choice(
+            "Saved file format",
+            [
+                "png",
```

Comment:
Usually these kinds of string constants for options are put at the top of the module
E.g. https://github.com/CellProfiler/CellProfiler/blob/master/cellprofiler/modules/labelimages.py#L53
This makes it so that if the strings ever change, they only have to be changed in one place